### PR TITLE
Corrected the TextRenderer parameter names

### PR DESCRIPTION
**Pull request by Sollace**
Corrected some wrong parameter names and filled in all the missing ones.

### CODE REVIEW

File: mappings/net/minecraft/client/font/TextRenderer.mapping [hunk 1]
```diff
@@ -27,18 +27,20 @@ CLASS net/minecraft/class_327 net/minecraft/client/font/TextRenderer
...
 	METHOD method_1712 drawTrimmed (Ljava/lang/String;IIII)V
-		ARG 1 string
+		ARG 1 text
 		ARG 2 x
 		ARG 3 y
 		ARG 4 maxWidth
+		ARG 5 color
 	METHOD method_1713 getStringBoundedHeight (Ljava/lang/String;I)I
 		ARG 1 str
+		ARG 2 maxWdth
```
Comment (Juuxel) [inline mappings/net/minecraft/client/font/TextRenderer.mapping]: There's a typo here.

**Review by Juuxel — CHANGES_REQUESTED** [review]

File: mappings/net/minecraft/client/font/TextRenderer.mapping (hunk 1 above)
Comment (Sollace) [reply mappings/net/minecraft/client/font/TextRenderer.mapping]: Should be fixed now. Thank ye.

**Review by Juuxel — APPROVED** [review]

**Review by YanisBft — APPROVED** [review]

**Review by liach — APPROVED** [review]

File: mappings/net/minecraft/client/font/TextRenderer.mapping [hunk 2]
```diff
@@ -50,30 +52,34 @@ CLASS net/minecraft/class_327 net/minecraft/client/font/TextRenderer
 		ARG 2 x
```
Comment (liach) [inline mappings/net/minecraft/client/font/TextRenderer.mapping:52]: Change that `str` above this line to `text` as well.

**Review by liach — CHANGES_REQUESTED** [review]

File: mappings/net/minecraft/client/font/TextRenderer.mapping (hunk 2 above)
Comment (Sollace) [reply mappings/net/minecraft/client/font/TextRenderer.mapping:52]: Oh, good catch!

Should really have all of them changed.

**Review by modmuss50 — APPROVED** [review]

**Pull request merged**